<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Pomodoro Timer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bPomodoro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Pomodoro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bShortBreak">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>10</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Short
Break</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bLongBreak">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Long
Break</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdMinute">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>120</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSecond">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>120</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lPomodoroText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Completed Sessions</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPomodoroCount">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bStartStop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lstatusBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>340</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background-color : rgba(70,142,145,255);}</string>
    </property>
    <property name="text">
     <string>Pomodoro Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
